### DISCUSSION TITLE
Add pronouns field to Agent table in schema

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#5971] [Feature Requests]
**Is your feature request related to a problem? Please describe.**

We don't currently have a field for this, and it'd be good to have this built-in to the schema for institutions to log this information for the agents in their database

**Describe the solution you'd like**

* A field `pronouns` to the agent table

**Comment by g1rly-c0d3r** [comment]
linked in https://github.com/specify/specify7/issues/8200
